src/test/burn/WixTestTools/RuntimePrereqFeatureFactAttribute.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+
+namespace WixTestTools
+{
+    using System;
+    using System.Linq;
+    using System.Management;
+    using System.Collections.Generic;
+    using System.Security.Principal;
+    using WixInternal.TestSupport.XunitExtensions;
+
+    public class RuntimePrereqFeatureFactAttribute : RuntimeFactAttribute
+    {
+        public static HashSet<string> OptionalFeatures = new();
+        public static HashSet<string> ServerFeatures = new();
```

Comment:
How about passing in `StringComparer.OrdinalIgnoreCase` to the hash sets so we don't have to worry about matching the case Windows chooses to use?

Reply:
done